### PR TITLE
Initial blog PR: "Using heuristic evaluation to inform the user exper…

### LINKED ISSUE #3077: Using heuristic evaluation to inform the user experience strategy of a product
**Describe the blog post your would like to write**
This blog will cover what a heuristic evaluation is. How it can benefit cloud tooling vs. a website and more, the way to conduct a heuristic evaluation and best practices offered to the community

**What is the title of the blog post?**
Using heuristic evaluation to inform the user experience strategy of a product

**Who are the authors?**
Ray and Aparna

**What is the proposed posting date?**
July 19th; write up is ready; see quip

### PR DESCRIPTION
**Pull request by nateynateynate**
…ience strategy of a product"

Includes new profile for one Ray Salas

### Description
Filing blog PR on @apasun 's behalf. 
 
### Issues Resolved
#3077 

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+![](/assets/media/blog-images/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product/Image.jpg)
+**Focus on workflows that help customers achieve a goal rather than complete specific tasks**. The Playground home page offers customers concrete tasks like ingesting data, exploring data, and trying a query assistant. Think about organizing the site to support customer goals such as proactive anomaly monitoring, visualizing your data, and getting started with minimal expertise.
+
+**Use consistent terminology and workflows across all tools and features to help customers work faster and build intuitive mental models.** There are several instances where the experience changes depending on whether users are logged in or not. There are also instances where terms like “buckets” are used that differ from industry standards. Simplifying terminology and workflows to follow industry knowledge and best practices will increase customer confidence that this product is for them.
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:83]:
```suggestion
1. **Use consistent terminology and workflows across all tools and features to help customers work faster and build intuitive mental models.** There are several instances where the experience changes depending on whether users are logged in or not. There are also instances where terms are used that differ from industry standards, like "buckets." Simplifying terminology and workflows to follow industry best practices will increase customer confidence in the product.
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+**Minimize available actions within the Playground that trial users cannot access**. A recurring Tenet & Trap violation was Inviting Dead End. Users encounter options (e.g. “add sample data”) within the Playground that they do not have access to and are unable to perform. Focusing on eliminating these encounters and errors make it easier for potential customers to assess the platform.
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:85]:
```suggestion
1. **Minimize available actions in OpenSearch Playground that trial users cannot access**. A recurring Tenet & Trap violation was Inviting Dead End. Users encounter options (for example, "add sample data") that they do not have access to and are unable to perform. Focusing on eliminating these encounters and errors makes it easier for potential customers to assess the product.
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+**Investigate potential content gaps on [OpenSearch.org](http://opensearch.org/) which** may hinder new user trial. The customer journey on [OpenSearch.org](http://opensearch.org/) seems to have information gaps that add friction to understanding and trialing of the Playground.
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:87]:
```suggestion
1. **Investigate potential content gaps on [OpenSearch.org](http://opensearch.org/)** that may hinder a new user trial. The website appears to have information gaps that add friction to understanding and trialing OpenSearch Playground.
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+
+## Takeaways
+
+* Perform a heuristics analysis to rapidly identify common traps and violations in your product design
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:92]:
```suggestion
* Perform a heuristic analysis to rapidly identify common traps and violations in your product design.
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+* You can scope your research as broadly or narrowly as you need
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:93]:
```suggestion
* You can scope your research as broadly or narrowly as needed.
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+* UX heuristics insights are often universally applicable to your entire UX; be sure to broaden your understanding (and dissemination) of your findings to ensure maximum value from your efforts
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:94]:
```suggestion
* UX heuristic insights are often universally applicable to your entire UX; be sure to broaden your understanding (and dissemination) of your findings to ensure maximum derived value from your efforts.
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+* You can engage outside UX research experts or utilize existing resources to apply these UI Tenets and Traps and 10 Usability Heuristics to your product
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:95]:
```suggestion
* You can engage outside UX research experts or utilize existing resources to apply these UI Tenets & Traps and Nielsen's 10 Usability Heuristics to your product.
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+* Implement a regular Heuristics research practice to monitor and track improvements.
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:96]:
```suggestion
* Implement a regular heuristic research practice to monitor and track improvements.
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+
+## References
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:98]:
```suggestion
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+
+[UI Tenets & Traps](https://uitraps.com/)
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:100]:
```suggestion
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+[Nielsen’s 10 Usability Heuristics](https://www.nngroup.com/articles/ten-usability-heuristics/)
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:101]:
```suggestion
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown (hunk 9 above)
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:98]: Deleted this section, as we already link to and explain these resources earlier in the blog post.

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 12]
```diff
@@ -0,0 +1,120 @@
...
+
+
+### About Steyer
+
+
+"We don't just analyze data, we create meaning from it. We don't just report insights, we inspire action from them."
+
+Steyer partners with our clients to make meaningful connections between people and information, solving our clients’ business problems with the tools we know best: strategic analysis, user research, and the creation, organization, and revision of business content. With nearly three decades of delighting both our clients and our own team members, we understand what it takes to effect real change in the real world.
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:109]:
```suggestion
Steyer partners with our clients to make meaningful connections between people and information, solving our clients' business problems with the tools we know best: strategic analysis, user research, and the creation, organization, and revision of business content. With nearly three decades of delighting both our clients and our own team members, we understand what it takes to effect real change in the real world.
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 13]
```diff
@@ -0,0 +1,120 @@
...
+
+![](/assets/media/blog-images/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product/Image1.jpg)
+
+
+**About Ray Salas - Steyer Insight Research Lead and Strategist**
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:114]:
```suggestion
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 14]
```diff
@@ -0,0 +1,120 @@
...
+
+
+As a leader with 20+ years of team building and people management experience at companies like JP Morgan Chase, Microsoft, and Amazon, Ray has been both a creator and consumer of customer research. As a research leader, he has implemented frameworks that deeply integrate human insight into engineering and product lifecycles, and he has driven prioritizations and roadmaps by balancing human insight, tech capability, and business viability. Above all, Ray believes in product and service development that is customer led and not just customer validated.
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:117]:
```suggestion
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown [hunk 15]
```diff
@@ -0,0 +1,120 @@
...
+
+![](/assets/media/blog-images/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product/Image2.jpg)
```
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:119]:
```suggestion
```

File: _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown (hunk 13 above)
Comment (natebower) [inline _posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown:114]: Deleted, as this information is already provided in rsalas.md. If the image is not already included in that file, please relocate.

**Review by natebower — COMMENTED** [review]
@apasun @pajuric Editorial review complete. Please see my changes and let me know if you have any questions. Thanks!

**Comment by pajuric** [comment]
@apasun - Please ping me when you have made the edits.

**Comment by nateynateynate** [comment]
@pajuric - I went ahead and committed all of @natebower 's suggestions. I think we're just waiting on meta now.

**Comment by pajuric** [comment]
@nateynateynate @apasun - Please add the following meta: 

_meta_keywords:_ UX heuristic evaluation, OpenSearch Plaground, OpenSearch UI design, Steyer Insights

_meta_description:_ Learn why the OpenSearch Project partners with Steyer Insights to conduct a heuristic evaluation of the OpenSearch Playground and how these insights are informing a refined getting started process.

**Review by krisfreedain — APPROVED** [review]

**Pull request merged**